andela/haven-ah-frontend
# #161342791 Setup an empty react project and host it on heroku

#### What does this PR do?
Setup an empty react project
#### Description of Task to be completed?
- setup and configure webpack
- configure babel to transpile to es5
- configure eslint with airbnb styleguide

#### How should this be manually tested?
- Git clone the repository
- cd into the `haven-ah-frontend` folder
- run `npm run start:dev`
- To see the production build visit [`Authors Haven App`](https://haven-ah-frontend.herokuapp.com/)

#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#161342791](https://www.pivotaltracker.com/story/show/161342791)
#### Screenshots (if appropriate)
<img width="1439" alt="screen shot 2018-11-26 at 3 25 18 pm" src="https://user-images.githubusercontent.com/34472497/49019974-92c80700-f18f-11e8-9551-c7f02485f009.png">
<img width="1440" alt="screen shot 2018-11-26 at 4 50 11 pm" src="https://user-images.githubusercontent.com/41582565/49025284-96fa2180-f19b-11e8-8ab2-617dd9f10607.png">


#### Questions:
N/A
